packages/p/protobuf-cpp/xmake.lua

```diff
@@ -38,15 +38,26 @@ package("protobuf-cpp")
         if package:config("zlib") then
             package:add("deps", "zlib")
         end
+        if package:version():gt("3.19.4") then
+            package:add("deps", "abseil", {configs = {shared = package:is_plat("windows")}})
```

Comment:
conflict can occur here if abseil is also added statically in project

Reply:
I was used shared for abseil since it resolved these jobs for windows: https://github.com/xmake-io/xmake-repo/actions/runs/8534520001

Reply:
We can apply patch to fix it, please do not pass shared config.